server/statistics/hot_peer_cache.go

```diff
@@ -48,6 +48,8 @@ var minHotThresholds = [RegionStatCount]float64{
 	RegionWriteKeys:  32,
 	RegionReadBytes:  8 * 1024,
 	RegionReadKeys:   128,
+	RegionReadQuery:  1, // temp
```

Comment:
What does `temp` here mean?

Reply:
it only is a temp threshold, I will obtain more meanable value by test